main.qml

```diff
@@ -414,6 +415,44 @@ ApplicationWindow {
         }
     }
 
+    function handleSweepUnmixable() {
+        console.log("Creating transaction: ")
+
+        transaction = currentWallet.createSweepUnmixableTransaction();
+        if (transaction.status !== PendingTransaction.Status_Ok) {
+            console.error("Can't create transaction: ", transaction.errorString);
+            informationPopup.title = qsTr("Error") + translationManager.emptyString;
+            informationPopup.text  = qsTr("Can't create transaction: ") + transaction.errorString
+            informationPopup.icon  = StandardIcon.Critical
+            informationPopup.onCloseCallback = null
+            informationPopup.open();
+            // deleting transaction object, we don't want memleaks
+            currentWallet.disposeTransaction(transaction);
+
+        } else if (transaction.txCount == 0) {
+            informationPopup.title = qsTr("No unmixable outputs to sweep") + translationManager.emptyString
+            informationPopup.text  = qsTr("No unmixable outputs to sweep") + translationManager.emptyString
+            informationPopup.icon = StandardIcon.Information
+            informationPopup.open()
```

Comment:
You need to reset the onCloseCallback here. 
`informationPopup.onCloseCallback = null`